README.md

```diff
@@ -15,23 +15,28 @@ This will be quick and dirty list for now.
 * Use meaningful, readable names for variables, functions and files. Don't try to save characters.
 * Follow the [Zen of Python](https://www.python.org/dev/peps/pep-0020/), regardless of the language you are using at the moment. English also counts as a language.
 * Use git and commit often, even on one-person projects.
+* Documentation is a sign of the quality of an API. It's easier to write it when the design is right.
```

Comment:
I'd reformulate this, so it doesn't look like you will attempt documentation at the very last step. Something like: 

Documentation proves your API: complexities, inconsistencies and naming issues tend to become apparent only when you attempt to document your API.